libxrdp/xrdp_rdp.c

```diff
@@ -827,6 +830,18 @@ xrdp_rdp_incoming(struct xrdp_rdp *self)
     g_strncpy(self->client_info.client_port,
               self->sec_layer->mcs_layer->iso_layer->trans->port,
               sizeof(self->client_info.client_port) - 1);
+
+    /* log TLS version and cipher when TLS is used */
+    /* TODO: client_addr, client_port is empty when IPv6 enabled */
+    if (iso->selectedProtocol > PROTOCOL_RDP)
```

Comment:
Even if it's not a TLS connection, you can still log it. Perhaps you should log selectedProtocol.